src/screens/AddItem/AddItemScreen.tsx

```diff
@@ -0,0 +1,72 @@
+import { Picker } from "@react-native-picker/picker";
+import { useState } from "react";
+import { View, StyleSheet, TextInput, Text, FlatList } from "react-native";
+
+interface addItemProps {
+  navigation: any;
+  route: any;
+}
+
+export default function AddItemScreen(props: addItemProps) {
+  const [selectedValue, setSelectedValue] = useState("");
+  const renderProductList = () => {
+    return Object.keys(props.route.params).map((key: string, index: number) => {
```

Comment:
This line in particular is wonky, the data we want is in props.route.params but that is an object not an array so we have to manipulate it a little strangely. Let me know if you guys have a better implementation for this.